examples/apache_httpd/BUILD

```diff
@@ -2,13 +2,15 @@ load("@rules_foreign_cc//tools/build_defs:configure.bzl", "configure_make")
 
 configure_make(
```

Comment:
Given that this was broken but CI continued to produce passing builds, I suspect this is not actually building in CI. Can you check to see that it's hooked up correctly?

Reply:
I don't know. Most of the packages in the `examples` workspace are not used in any way by the CI

Reply:
I think it'd be simple enough to add `apache_httpd` to [this list](https://github.com/bazelbuild/rules_foreign_cc/blob/87df6b25f6c009883da87f07ea680d38780a4d6f/examples/BUILD#L19-L33) to make sure the target is building in CI.

Reply:
I will try

Reply:
I tried on Linux and Mac. Mac fail due to 
```
configure: error: could not determine /private/var/tmp/_bazel_buildkite/8afef7f0076038b0098d9c7983f77386/sandbox/darwin-sandbox/196/execroot/rules_foreign_cc_tests/external/local_config_cc/libtool interface
```
so I have switched to Linux only test suite

Reply:
I think that's fine for the time being. Unfortunate though